### PR TITLE
VCR recordings for ARM azure_mgmt_network tests

### PR DESCRIPTION
**Pull request by vishrutshah**
Adding support for VCR recording in ARM azure_mgmt_network service

This changes is part of the feature #291

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vishrutshah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: resource_management/.123env [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# This is a sample of what your .env file should look like if you wanted to run integration tests.
```
Comment (veronicagg) [inline resource_management/.123env]: should this file name be .env_sample as the one in asm just to be consistent?

**Comment by veronicagg** [comment]
@vishrutshah Left a comment, rest looks good! thanks.

File: resource_management/.123env (hunk 1 above)
Comment (vishrutshah) [inline resource_management/.123env]: Good catch. This file should not be here at all. As you mentioned we already have .env_smaple. By mistake it got committed.

**Comment by veronicagg** [comment]
@vishrutshah looks good, thanks!

**Pull request merged**